### ISSUE TITLE
Zap iOS not loading wallet with Raspberry 4 8GB RAM

### ISSUE THREAD

**Issue by Javihache** [Stadicus/RaspiBolt#698]
I have Ubuntu 64 on a Raspberry Pi 4 Model B 8GB. This is a similar problem I had previously with my Raspberry Pi 2 1GB and my LND + Bitcoind node in docker. I thought it might be the Raspberry Pi 2 being too old, that's the reason why I actually updated to a Pi 4, but the problem is still there.

I followed the procedure and the node works and is synced with graph and chain. As and alternative I tried to connect Joule Wallet from Chrome (which uses the REST port instead of gRPC). Joule works and connects to the wallet.

However once I add my node to Zap, it just stays loading for a few seconds (exactly 15s) and then it just goes back to the menu where the added nodes are listed. 

LND received the request and the logs shows the entry: `[INF] INVC: New invoice subscription client: id=1` Therefore I assume the request from Zap arrives correctly to the Pi.

It seems to me this might be a timeout problem, but I don't get why the Raspberry Pi takes so long to send whatever data Zap Wallet requires. Are you guys seeing a similar problem? Is there anything I should adjust to solve this problem either on the Raspberry Pi itself, my network or lnd?

Forgot to mention: I am using bitcoind 0.21.0 and lnd 0.12.0rc5

**Comment by Javihache** [comment]
I just tried to connect with Zap Desktop and I just get "Unable to connect to host: 4 DEADLINE_EXCEEDED: Deadline exceeded. I am guessing it really is a timeout problem.

**Comment by Javihache** [comment]
I just realized that the lnd.conf as per the documentation on https://stadicus.github.io/RaspiBolt/raspibolt_40_lnd.html is fitted with the entry `listen=localhost`. That's the reason why my devices couldn't access the node using gRPC and port 9735. I changed the lnd.conf to `listen=0.0.0.0:9735` and opened the port on my router. Now everything works fine! I will leave this here in case someone else has the same problem!
